DiscordBot/Commands/EventHelper.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DiscordBot.DataAccess;
+using DiscordBot.DataAccess.Exceptions;
+using DiscordBot.DataAccess.Models;
+using DSharpPlus.Entities;
+
+namespace DiscordBot.Commands
+{
+    class EventHelper
```

Comment:
So how is this class different to EventCommands? Each class should have a clear responsibility and I'm not entirely sure what the responsibility of this class is? 🙂

Reply:
The methods in EventCommands aren't static and I can't reuse the methods within it without instantiating the class, which I don't think I'm supposed to do as the class is normally created from the API's RegisterCommands method.

Reply:
Or I guess I could have made those methods static within EventCommands? I probably just didn't think of that.

Reply:
Ah okay, so these are just methods that need to be used both by the EventCommands and by the OnMessageReactionAdded method? That's probably fine. It would have been nice with a cleaner split of concerns but we don't want to make EventCommands too big either. So it's probably fine like this as long as you make sure they're using the same service instance 🙂

Reply:
I was also thinking about extracting some methods from EventCommands into the helper method since EventCommands is quite big, what do you think?

Reply:
And since I'm relying on static methods I suppose I should pass the service as a parameter?

Reply:
Do the methods need to be static? Can the event handler method be an instance method?